build/Toolset.proj

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
```

Comment:
Is this the right TFM? Will it work crossplat?

Reply:
Its working everywhere so-far. Possibly because repo-toolset is a `tools` type project, where NuGet isn't doing any validation.